src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -20,32 +20,33 @@ To update an existing version of Flutter, see [Upgrading Flutter](/docs/developm
 
 ### Update your path
 
-If you wish to run Flutter commands in the regular Windows command prompt, take
+If you wish to run Flutter commands in the regular Windows console, take
 these steps to add Flutter to the `PATH` environment variable:
 
-* Go to **Control Panel > User Accounts > User Accounts > Change my environment
-  variables**
+* From the Start search bar, type 'env' and select **Edit environment
+  variables for your account**
 * Under **User variables** check if there is an entry called **Path**:
   * If the entry does exist, append the full path to `flutter\bin` using `;` as
     a separator from existing values.
   * If the entry does not exist, create a new user variable named `Path` with
     the full path to `flutter\bin` as its value.
 
-Reboot Windows to fully apply this change.
+Note that you will have to close and reopen any existing command prompt windows 
```

Comment:
Note that you use "command prompt windows" here but "console" above and below.  Whatever terminology we choose, we should be consistent.

Reply:
@jamesderlin, Tim tested this on a Windows box, so I figured the terminology was right. I agree with your point about consistency, though.

Reply:
See comments below. You're right -- `console windows` would be better here.